### PR TITLE
use css to change background-image of logo for pixelate effect

### PR DESCRIPTION
**Pull request by lukey-luke**
This PR aims to address #11.

In order to perform this pixelate effect without the use of javascript, background-image is set on `:hover`. Note that there appears to be some quality degradation when using the image this way. However, for the main text (no hover) the logo is made w/ text in the browser, which appears to provide the highest level of quality compared to `` or `css background-image` options

### CODE REVIEW

**Comment by etanb** [comment]
Screenshot 2024-07-03 at 2 06 09 PM

While the raw image you have looks fine, on the site itself, it looks "blurry" in a way and doesn't totally match up with the corresponding text underneath -- might be worth asking Jenn or Matt to formally make a pixelated version using the correct font. Also we'd probably want to do the same thing to the Verdance logo in the footer!

Last note: there's a loading pause when the image is hovered over and switched -- we'd want to pre-load the other image so there isn't a flash of no content.

File: src/assets/styles/_shared.scss [hunk 1]
```diff
@@ -440,3 +440,18 @@ h3 {
...
+  background-position-y: center;
+  background-repeat: no-repeat;
+  background-size: contain;
+
+  &:hover {
+    cursor: pointer;
+    // background-color: skyblue;
+    // color: transparent; 
+    // text-shadow: 0 0 3px rgba(0,0,0,0.5); 
+    background-image: url('/assets/img/logos/Verdance-logo-pixelated-black.png');
```
Comment (lukey-luke) [inline src/assets/styles/_shared.scss:455]: browsers might not load this `.main-logo:hover` image right away, causing delay to show pixelate effect.
There are probably a number of ways to achieve desired behavior. If you request the `` w/ `display=none` request should still be made and could be displayed or the cache version of the image could be used in the `background-image url()`

Comment (etanb) [reply src/assets/styles/_shared.scss:455]: you can also use the `preload` option within `link`: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/preload

Comment (lukey-luke) [reply src/assets/styles/_shared.scss]: Cool, I didn't know about `preload`!

I couldn't get rid of the blurriness regardless of using `` or `background-image: url()` for the Verdance logo text. See below.

Screenshot 2024-07-08 at 12 58 42 PM

In order to work around this, I left the main logo text in `<span>Verdance</span>`, but changed visibility + background-image for hover

**Comment by amprokop** [comment]
image

**Comment by amprokop** [comment]
image

**Comment by amprokop** [comment]
The only issue I see here is that there's a pretty brief loading flash the first time you hover over the logo. Is that onerous to fix? I'm not seeing any blurriness in my browser. @lukey-luke @etanb

**Comment by lukey-luke** [comment]
closing, this was resolved in https://github.com/VerdanceTeam/verdance-site/pull/37

**Pull request closed without merging**